Spectral[3] spectrals